app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,56 @@
+class FoodsController < ApplicationController
```

Comment:
- [ ] I can create new food without logging in, but when saving I get an error. I should not be able to do that
![Screenshot from 2022-04-30 17-12-07](https://user-images.githubusercontent.com/62705947/166111384-eec8b957-f194-45a6-9f18-36e68e17255f.png)

Reply:
Thank you for the review. Changes have been made. Logged out users are no more able to add food